CHANGELOG.md

```diff
@@ -2,4 +2,36 @@
 
 IMPROVEMENTS:
 
-* The zone ID and the DNS name of the ELB are now output from the module.   
\ No newline at end of file
+* The zone ID and the DNS name of the ELB are now output from the module.   
+
+## 0.1.5 (November 27 2020)
+
+IMPROVEMENTS:
+
+* Added a new boolean `deploy_in_vpc` flag to disable VPC deployments.
```

Comment:
I usually use the values `"yes"` and `"no"` rather than boolean true / false for these options. The reason being that I wrote the first infrablocks modules back when terraform only supported string variables.

Would it be possible to make this module / variable consistent with that approach?